prototype2/js/model.js

```diff
@@ -82,11 +82,15 @@ class Model {
 
     /**
      * Return an array of colors that represent different level in the map/legend based on
-     * the darkest color
+     * the darkest and lightest color
      * @param {*} maxColor
      */
-    interpolate(maxColor) {
-        let colors = [];
+    interpolate(maxColor, minColor) {

```

Comment:
I think it's more common to see the order minColor, maxColor (low to high) than high to low.